mods/default/nodes.lua

```diff
@@ -586,29 +586,129 @@ minetest.register_node("default:snow", {
 	inventory_image = "default_snowball.png",
 	wield_image = "default_snowball.png",
 	paramtype = "light",
+	paramtype2 = "leveled",
 	buildable_to = true,
 	floodable = true,
+	walkable = true,
 	drawtype = "nodebox",
+	leveled = 7,
```

Comment:
```suggestion
	leveled = 8,
```
It might also make sense to set a placement_param2 and to disable placement prediction.

Reply:
Will do.

Reply:
Yes 8.